docs/eventing/samples/apache-camel-source/README.md

```diff
@@ -11,64 +10,71 @@ for generating events.
    any namespace where you want to run Camel sources.
 
    The preferred version that is compatible with Camel sources is
-   [Camel K v0.2.0](https://github.com/apache/camel-k/releases/tag/0.2.0).
+   [Camel K v0.3.3](https://github.com/apache/camel-k/releases/tag/0.3.3).
 
    Installation instruction are provided on the
    [Apache Camel K Github repository](https://github.com/apache/camel-k#installation).
    Documentation includes specific instructions for common Kubernetes
    environments, including development clusters.
 
-1. Install the Camel Source from the `camel.yaml` in the
-   [Eventing Sources release page](https://github.com/knative/eventing-contrib/releases):
+1. Install the Camel Source from the [yaml files in the config dir](../config/)
+   from source:
 
    ```shell
-   kubectl apply --filename camel.yaml
+   ko apply -f ../config/
```

Comment:
If you switch over to `ko`, I think it should be added before this command in the "prerequisites" section

Reply:
I think the previous version is good for this general doc. Using ko is something for devs that work on the knative-contrib repo